pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/stream/EOFException.java

```diff
@@ -0,0 +1,10 @@
+package com.hedera.pbj.runtime.io.stream;
+
+import java.nio.BufferUnderflowException;
+
+/** This class is used as an exception to signal that the end of stream is reached
+ * when reading.
+ */
```

Comment:
minor ;-) would be nice to be one of these styles
```
/** 
  * This class is used as an exception to signal that the end of stream is reached
  * when reading.
  */
```
or
```
/**  This class is used as an exception to signal that the end of stream is reached when reading.  */
```

Reply:
Changed the comment. It is less than 120 chars.